File: ws/hidalgo/u2/ws12inheritance/FarmSystem/src/main/java/ec/edu/espe/farmsystem/model/Cage.java
package ec.edu.espe.farmsystem.model;

import javax.tools.DocumentationTool.Location;
import javax.tools.JavaFileManager;

/**
 *
 * @author Steven Loza
 */
public class Cage {

    private int id;
    private String description;
    private int type; // 1 coop, 2 stable, 3 pens
    private Location location;

    public Cage(int i, String satble_for_cows, int i0, JavaFileManager.Location Location) {
   
    }

    @Override
    public String toString() {
        return "Cage{" + "id=" + id + ", description=" + description + ", type=" + type + ", location=" + location + '}';
    }

    public Cage(int id, String description, int type, Location location) {
        this.id = id;
        this.description = description;
        this.type = type;
        this.location = location;
    }

    
    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * @param description the description to set
     */
    public void setDescription(String description) {
        this.description = description;
    }

    /**
     * @return the type
     */
    public int getType() {
        return type;
    }

    /**
     * @param type the type to set
     */
    public void setType(int type) {
        this.type = type;
    }

    /**
     * @return the location
     */
    public Location getLocation() {
        return location;
    }

    /**
     * @param location the location to set
     */
    public void setLocation(Location location) {
        this.location = location;
    }

   

}

File: hw/bonilla/u2/hw19Inheritance/FarmSystemPython/model/Cage.py
class Cage:
    def __init__(self, id, description, type, location):
        self.id = id
        self.description = description
        self.type = type
        self.location = location

    def __str__(self):
        return f"""{{
        "id": {self.id},
        "description": "{self.description}",
        "type": {self.type},
        "location": {self.location}
    }}"""

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_description(self):
        return self.description

    def set_description(self, description):
        self.description = description

    def get_type(self):
        return self.type

    def set_type(self, type):
        self.type = type

    def get_location(self):
        return self.location

    def set_location(self, location):
        self.location = location
